routing_model/src/utilities.rs

```diff
@@ -74,6 +74,9 @@ pub struct SectionInfo(pub Section, pub i32 /*contain full membership */);
 #[derive(Debug, Clone, Copy, Default, PartialEq, PartialOrd, Ord, Eq)]
 pub struct GenesisPfxInfo(pub SectionInfo);
 
+#[derive(Debug, Clone, Copy, Default, PartialEq, PartialOrd, Ord, Eq)]
+pub struct NeighbourMergeInfo;
```

Comment:
Maybe call it MergeInfo, because it comes from the Merge RPC

Reply:
Done